clients/admin-ui/src/features/common/form/ControlledRadioGroup.tsx

```diff
@@ -0,0 +1,115 @@
+import type { AntRadioGroupProps as RadioGroupProps } from "fidesui";
+import {
+  AntFlex as Flex,
+  AntRadio as Radio,
+  FormControl,
+  Grid,
+  RadioChangeEvent,
+  Text,
+} from "fidesui";
+import { useField } from "formik";
+
+import type { StringField } from "~/features/common/form/inputs";
+import { ErrorMessage, Label, Option } from "~/features/common/form/inputs";
+import QuestionTooltip from "~/features/common/QuestionTooltip";
+
+interface ControlledRadioGroupProps extends RadioGroupProps {
+  label?: string;
+  options: Option[];
+  layout?: "inline" | "stacked";
+  defaultFirstSelected?: boolean;
+}
+
+const ControlledRadioGroup = ({
+  label,
+  options,
+  layout,
+  defaultFirstSelected = true,
+  ...props
+}: ControlledRadioGroupProps & StringField) => {
+  const [initialField, meta] = useField(props);
+  const field = { ...initialField, value: initialField.value ?? "" };
+  const isInvalid = !!(meta.touched && meta.error);
+  const defaultSelected = defaultFirstSelected ? options[0] : undefined;
+  const selected =
+    options.find((o) => o.value === field.value) ?? defaultSelected;
+
+  const handleChange = (e: RadioChangeEvent) => {
+    field.onChange(props.name)(e.target.value);
+  };
+
+  if (layout === "stacked") {
+    return (
+      <FormControl isInvalid={isInvalid}>
+        <Flex className="w-fit">
+          {label ? (
+            <Label htmlFor={props.id || props.name}>{label}</Label>
```

Comment:
This label doesn't actually tie to any form elements, so we should just leave off the `htmlFor` altogether. It's mostly a decorative label.